From 5b4ba173f6a84a8be1131b1523dc42df95171ec7 Mon Sep 17 00:00:00 2001
From: Antonio Rojas <arojas@archlinux.org>
Date: Thu, 20 Apr 2023 17:49:44 +0200
Subject: [PATCH] Fix build with akonadi 23.04

In KDE PIM 23.04 the KF5:: targets have been renamed to KPim5::
While the KF5:: targets are kept for backwards compatibility, they no longer recursively include other KF5:: targets.
We need to search for Akonadi explicitly so that the KF5::AkonadiCore target is defined.
---
 core/CMakeLists.txt | 1 +
 1 file changed, 1 insertion(+)

diff --git a/core/CMakeLists.txt b/core/CMakeLists.txt
index 4cb33d2f9e6..610033425cd 100644
--- a/core/CMakeLists.txt
+++ b/core/CMakeLists.txt
@@ -124,6 +124,7 @@ if(ENABLE_AKONADICONTACTSUPPORT)
 
     find_package(KF5 ${AKONADI_MIN_VERSION} QUIET REQUIRED
                  OPTIONAL_COMPONENTS
+                 Akonadi
                  AkonadiContact         # For KDE Mail Contacts support.
                  Contacts               # API for contacts/address book data.
     )
-- 
GitLab

